build.cmd

```diff
@@ -49,6 +49,8 @@ if /i "%1" == "vs2015" (set __VSVersion=%1&shift&goto Arg_Loop)
 if /i "%1" == "skiptestbuild" (set __SkipTestBuild=1&shift&goto Arg_Loop)
 if /i "%1" == "docrossgen" (set __DoCrossgen=1&shift&goto Arg_Loop)
 
+if /i "%1" == "priority" (set _priorityvalue=%2&shift&shift&goto Arg_Loop)
```

Comment:
I think this should be called testpriority or something. Also, I'd start the variable name with two underscores and camel case it to match the convention here.